spec/classes/mod/fcgid_spec.rb

```diff
@@ -38,15 +36,13 @@
         :id                        => 'root',
         :kernel                    => 'Linux',
         :path                      => '/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin',
-        :is_pe                     => false,
+        :is_pe                  => false,
       }
```

Comment:
I try to fix just one think in this PR. Mixing things together leads to regressions.

But I will add a commit for this indentation fix.